### DISCUSSION TITLE
Is there a way to hot reload changes?

### DISCUSSION THREAD

**Discussion by arkanmgerges** [structurizr/lite#38] [Q&A]
Hi,

I'm using docker command `docker run --rm -v $(pwd):/usr/local/structurizr -p 8080:8080  structurizr/lite` is there any way to hot reload the changes using plugins (jetbrains), extensions (vscode). I tried "live server" with proxy but it does not reload the changes done to `workspace.dsl`.
If there are any way to do it, please shed a light on it.

Thanks

**Comment by arkanmgerges** [comment]
I tried also "Auto refresh page" extension for fire fox, but it seems like ugly reload, as it reload all the page. I need something to reload the content not all the page top keys, like this part 
![image](https://user-images.githubusercontent.com/7836805/222922278-37ab8b30-94b7-4c35-b6e2-09c0b2c2a7f0.png)

**Comment by simonbrowndotje** [comment]
Structurizr Lite's built-in auto-refresh feature will also do an "ugly reload", so I'm not sure what else to suggest I'm afraid.

**Comment by arkanmgerges** [comment]
Thank you @simonbrowndotje 

I'm using the auto-refresh now (https://github.com/structurizr/lite/blob/main/docs/docs/04-usage.md#auto-refresh).
I wish if there was something like reactjs hot reload, flutter hot reload, or solidjs hot reload, rather than reload all the page (and it's also slower compared with what I mentioned). But if this is hard to do and also it's not a high priority or something that is blocker. It is better than manually refreshing the page.

Best regards,
Arkan
